librz/core/cmd/cmd_eval.c

```diff
@@ -53,7 +53,92 @@ static bool pal_seek(RzCore *core, RzConsPalSeekMode mode, const char *file, RzL
 	return true;
 }
 
-RZ_API bool rz_core_theme_load(RzCore *core, const char *name) {
+RZ_API bool rz_core_theme_print_metainfo(RzCore *core, RzCmdStateOutput *state, RzThemeInfo *meta_info) {
+	bool failed = false;
+
+	if (meta_info->author != NULL) {
```

Comment:
You could just use:
```c
rz_cons_printf("Author: %s\n", rz_str_get(meta_info->author));
```
It will take care fo the NULL, add a newline, also mark it's the author.